src/components/form-editor/field-editor/FieldEditor.tsx

```diff
@@ -304,13 +348,15 @@ const FieldEditor = ({
                     autoFocus={i === fieldFormData.options.length - 1}
                     className={classes.input}
                     fullWidth
-                    id={`${field.id} option ${i}`}
+                    id={`${getId()} option ${i}`}
+                    inputProps={{ maxLength: 64 }}
                     onChange={(e) =>
                       onUpdateOption({
                         index: option.index,
                         value: e.target.value,
                       })
                     }
+                    placeholder="Option name"
```

Comment:
First options defaults to Option 1, rest default to this placeholder. Just wanted to point it out.
![Screen Shot 2021-08-30 at 6 31 47 PM](https://user-images.githubusercontent.com/21024100/131414315-5fc76279-d148-4eff-8b6b-5a033ae4e8e1.png)

Reply:
will try to make this default to "Option 2", "Option 3", etc. to follow other form builders more closely